biz.aQute.bnd/test/aQute/bnd/junit/JUnitFrameworkTest.java

```diff
@@ -0,0 +1,150 @@
+package aQute.bnd.junit;
```

Comment:
If the new class is in bndlib, why is the test for it in bnd? That seems wrong. It should be in bndlib.tests